src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -0,0 +1,266 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Data;
+using System.Windows.Documents;
+using System.Windows.Input;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
+using System.Windows.Navigation;
+using System.Windows.Shapes;
+using AccessibilityInsights.SharedUx.ViewModels;
+using AccessibilityInsights.Extensions.Helpers;
+using AccessibilityInsights.CommonUxComponents.Dialogs;
+using AccessibilityInsights.SharedUx.FileIssue;
+using System.IO;
+using AccessibilityInsights.Extensions.Interfaces.IssueReporting;
+using AccessibilityInsights.SharedUx.Telemetry;
+using AccessibilityInsights.SharedUx.Enums;
+
+namespace AccessibilityInsights.SharedUx.Controls.CustomControls
+{
+    /// <summary>
+    /// Interaction logic for ScannerResultCustomListControl.xaml
+    /// </summary>
+    public partial class ScannerResultCustomListControl : UserControl
+    {
+        private ScannerResultCustomListContext _controlContext;
+
+        public IEnumerable ItemsSource 
+        {
+            get => lvDetails.ItemsSource;
+            set { lvDetails.ItemsSource = value; } 
+        }
+
+        public int SelectedIndex { get; set; }
```

Comment:
This feels disconnected. Should it connect to lvDetails.SelectedIndex? Once we add a 2nd list, we'll need some sort of exclusive selection mechanism. That feels like the role of the parent, since the children don't know about each other

Reply:
It makes sense to me to connect this to lvDetails, and looking at the code, it doesn't seem like is going to be a lot of effort to implement a mechanism to make an exclusive selection since this is method is only referenced in one part of the code in the parent ScannerResultControl inside the UpdateTree() method.